web/client-api/src/main/java/io/deephaven/web/client/api/ConnectOptions.java

```diff
@@ -65,5 +68,8 @@ public ConnectOptions(Object connectOptions) {
         // if (map.has("fetch")) {
         // fetch = map.getAsAny("fetch").uncheckedCast();
         // }
```

Comment:
This comment could be removed.

Reply:
Could be, though we thought we would want to have it - in theory this is a simpler version of what we just implemented.

If we do decide we'll never want it, I think I'd want to expose canonical factories for fetch and websocket, so users can just assign them if they want to direct which built-in factory to use.